presentation/src/main/java/com/yapp/presentation/ui/AttendanceScreen.kt

```diff
@@ -92,6 +94,12 @@ fun AttendanceScreen(
         composable(
             route = AttendanceScreenRoute.HELP.route
         ) {}
+
+        composable(
+            route = "team"
```

Comment:
```suggestion
            route = AttendanceScreenRoute.SIGNUP.route
```
이 화면에 이름 선택, 팀 선택이 둘다 들어가지요? 로그인 직후 최초 한번만 하는 행위이니 sign up으로 짓는게 어떨까용?

Reply:
이름 선택과 팀 선택 찢을 것 같습니다! 흠... `SIGNUPNAME`과 `SIGNUPTEAM`은 ㅇ떤가요..?